### PR TITLE
Add test code for modules/canbus/vehicle/gem/protocol/wheel_speed_rpt_7a.cc

### PR DESCRIPTION
**Pull request by JasonXu12**

### CODE REVIEW

**Comment by JasonXu12** [comment]
@Capri2014, please review.

File: readme.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hi! This is Jason!
```
Comment (Capri2014) [inline readme.txt]: Please remove the readme.txt

File: modules/canbus/vehicle/gem/protocol/BUILD [hunk 2]
```diff
@@ -39,6 +39,7 @@ cc_library(
     ],
     hdrs = [
       "accel_cmd_67.h",
+
```
Comment (Capri2014) [inline modules/canbus/vehicle/gem/protocol/BUILD]: remove this extra blank line

**Comment by Capri2014** [comment]
@JasonXu12 Commit format start with "[Module Names] : [Commit contents]", for instance, for this one, change the commit/pull request to "Canbus : Add  test for gem wheel_speed_rpt_7a"

File: readme.txt (hunk 1 above)
Comment (JasonXu12) [reply readme.txt]: OK！

**Comment by Capri2014** [comment]
@JasonXu12 fix the lint issue please

**Comment by JasonXu12** [comment]
@Capri2014 solve the lint issue. please review. Thanks!

**Pull request merged**